src/modules/gateway/transak/Transak.ts

```diff
@@ -16,29 +15,15 @@ const PURCHASE_EVENT = 'Purchase status change'
 
 export class Transak {
   private readonly config: TransakConfig
+  private readonly customizationOptions: Partial<CustomizationOptions>
   private sdk: TransakSDK
 
-  constructor(config: TransakConfig) {
+  constructor(
+    config: TransakConfig,
+    customizationOptions?: Partial<CustomizationOptions>
+  ) {
     this.config = config
-  }
-
-  protected defaultCustomizationOptions(
-    address: string
-  ): DefaultCustomizationOptions {
-    return {
-      apiKey: this.config.key, // Your API Key
-      environment: this.config.env || 'STAGING', // STAGING/PRODUCTION
-      networks: 'ethereum,matic',
-      walletAddress: address, // Your customer's wallet address
-      hostURL: window.location.origin,
-      widgetHeight: '650px',
-      widgetWidth: '450px'
-    }
-  }
-
-  protected customizationOptions(address: string): CustomizationOptions {
-    return {
-      ...this.defaultCustomizationOptions(address),
+    this.customizationOptions = customizationOptions || {
```

Comment:
If customizationOptions is provided 

```
defaultCryptoCurrency: 'MANA',
cyptoCurrencyList: 'MANA',
fiatCurrency: '', // INR/GBP
```

Will not be set.

I recommend just setting an empty object on the constructor and adding those 3 to the defautlOptions

Reply:
They are not default. They are attached to the MANA purchases.

Reply:
They should be provided via constructor when doing mana purchases don't you think?